### PR TITLE
Update Getting Started Doc With Custom Kubeadm Instructions

### PR DESCRIPTION
**Pull request by james65535**
**What this PR does / why we need it**:

This documentation update provides steps on how to use a custom kubeadm configuration when provisioning workload clusters.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -312,3 +312,52 @@ Now that you have the `kubeconfig` for your Workload Cluster, you can start depl
 **NOTE**: workload clusters do not have any addons applied aside from those added by kubeadm. Nodes in your workload clusters
 will be in the `NotReady` state until you apply a CNI addon. The `addons.yaml` file generated from `make prod-yaml` has a default calico
 addon which you can use, otherwise apply custom addons based on your use-case.
+
+## Creating Workload Clusters with custom kubeadm configurations
+
+When provisioning new workload clusters, users may wish to customize aspects of their cluster with kubeadm configuration options other than those created by the default configuration.
+The `cluster.spec.providerSpec.value.clusterConfiguration` field allows for the inclusion of kubeadm configuration parameters via yaml.
+An example of the previous cluster configuration yaml file is shown below which includes the kubeadm configuration options.
+Please note, some kubeadm configuration fields are overridden by the provider and cannot be defined, for example the kubeadm networking fields.
```
Comment (andrewsykim) [inline docs/getting_started.md]: I'd take this a step further and configure something on the cluster outside the default, something like a k8s feature gate should be easier to follow https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ what do you think?

**Comment by james65535** [comment]
This PR depends on #389 
/hold

File: docs/getting_started.md (hunk 1 above)
Comment (james65535) [reply docs/getting_started.md]: Thanks Andrew, I've updated with examples of specifying an admission controller policy for cluster configuration and a node label for init configuration.

**Comment by james65535** [comment]
PR #389 merged, this request is ready to go.
/hold cancel

**Comment by andrewsykim** [comment]
@james65535 can you rebase this please?

**Pull request closed without merging**